packages/frontend/src/utils/project/getProjectTvlTooltipText.ts

```diff
@@ -0,0 +1,48 @@
+import { Layer2 } from '@l2beat/config'
+import { ValueType } from '@l2beat/shared-pure'
+
+export function getProjectTvlTooltipText(config: Layer2['config']) {
+  const hasCanonical = config.escrows.length > 0
+  const hasExternal = config.tokenList?.some(
+    (token) => token.type === ValueType.EBV,
+  )
+  const hasNative = config.tokenList?.some(
+    (token) => token.type === ValueType.NMV,
+  )
+  if (
+    (!hasCanonical && !hasExternal && !hasNative) ||
+    (hasCanonical && !hasExternal && !hasNative)
+  ) {
```

Comment:
```suggestion
  if (!hasExternal && !hasNative) {
```
The hasCanonical can be either true or false so we can skip it

Reply:
d2d382d